### ISSUE TITLE
Correct `fetch_Iowa_points`

### ISSUE THREAD

**Issue by padilla410** [USGS-R/lake-temperature-model-prep#316]
[`fetch_Iowa_points`](https://github.com/USGS-R/lake-temperature-model-prep/blob/42062e503a499b46b027a748912a0b36ea1f150d/1_crosswalk_fetch/src/fetch_crosswalk.R#L260-L272) converts UTM 15 (CRS = 26915) to lat/long (CRS = 4326). This conversion works for _most_ lakes in [`Iowa_Lake_Locations.csv`](https://drive.google.com/file/d/1k1U65PtXpb2X-blbPKBFJGbfG_tzbOjf/view?usp=sharing) except 3 locations which are zone 14 (CRS = 26914):

<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<body link="#0563C1" vlink="#954F72">

LakeID | Lake Name | County | Zone | UTM (NAD83)_E | UTM (NAD83)_N | STORET ID
-- | -- | -- | -- | -- | -- | --
12 | Blue Lake | MONONA | 14T | 735109 | 4658584 | 22670002
15 | Browns Lake | WOODBURY | 14T | 720509 | 4687502 | 22970001
68 | Lake Pahoja | LYON | 14T | 704889 | 4806432 | 22600001

</body>

</html>

Blue Lake is on the west side of Iowa:
![image](https://user-images.githubusercontent.com/15007294/157126636-10b84645-a110-4049-ad1c-78fdecb31948.png)

However, the current version of `fetch_Iowa_points` puts it on the east side of the state:
![image](https://user-images.githubusercontent.com/15007294/157127007-1e50d8a1-7356-4865-afd7-b16014084370.png)

```r
> library(scipiper)
> library(dplyr)
> 
> readRDS(sc_retrieve('1_crosswalk_fetch/out/Iowa_lake_pts_sf.rds.ind')) %>% 
+   filter(site_id == 'Iowa_12')
Simple feature collection with 1 feature and 5 fields
Geometry type: POINT
Dimension:     XY
Bounding box:  xmin: -90.15928 ymin: 42.04416 xmax: -90.15928 ymax: 42.04416
Geodetic CRS:  WGS 84
# A tibble: 1 x 6
  site_id `Lake Name` County Zone  `STORET ID`             geometry
* <chr>   <chr>       <chr>  <chr> <chr>                <POINT [°]>
1 Iowa_12 Blue Lake   MONONA 14T   22670002    (-90.15928 42.04416)
```

`fetch_Iowa_points` needs to be modified to capture these zone 14 lakes.

This issue was discovered when working on #286 and will be resolved in the same pull request.

**Comment by padilla410** [comment]
Here is a map of the UTM zones in Iowa (with hand annotation):
![image](https://user-images.githubusercontent.com/15007294/157245852-ad1fa0a3-8ed1-42b5-8c30-b72932889193.png)

[https://mangomap.com/robertyoung/maps/69585/what-utm-zone-am-i-in-#](https://mangomap.com/robertyoung/maps/69585/what-utm-zone-am-i-in-#)

**Comment by padilla410** [comment]
This should have closed with PR #321
